parts/k8s/addons/1.15/calico.yaml

```diff
@@ -41,7 +41,7 @@ data:
 
 ---
 {{- /* Source: calico/templates/kdd-crds.yaml */}}
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1{{- if not (IsKubernetesVersionGe "1.19.0")}}beta1{{end}}
```

Comment:
We can omit these conditionals if `apiextensions.k8s.io/v1` was supported on Kubernetes 1.15.

Reply:
This calico spec will only be delivered in a Kubernetes v1.15 cluster creation context, do we need to do anything here?

Reply:
Oops...bad example. I reverted changes to this file, but there are many similar changes in other manifests.